numpy/random/mtrand/mtrand.pyx

```diff
@@ -3837,20 +3837,19 @@ cdef class RandomState:
         -------
         out : ndarray or scalar
             Drawn samples from the parameterized negative binomial distribution,
-            where each sample is equal to N, the number of trials it took to
-            achieve n - 1 successes, N - (n - 1) failures, and a success on the,
-            (N + n)th trial.
+            where each sample is equal to N, the number of successes it took to
```

Comment:
This doesn't make sense.

Reply:
Fixed, thanks!

Reply:
Should this be "the number of trials it took"?

Reply:
The one in the most recent comment is correct and matched with Wikipedia. Trials = successes + failures. According to my unscientific test, the mean and variance of a large number of samples match up with what I wrote.

Reply:
I think my concern is more with the "it took" part.  You don't need successes in order to get failures, so that sentence construction reads strangely.  Perhaps something akin to "the number of successes that occurred before n failures"?

Reply:
That's much clearer, thanks for the suggestion!